SchibstedAccount/Manager/User/User.swift

```diff
@@ -49,6 +49,8 @@ public class User: UserProtocol {
     ///
     public weak var delegate: UserDelegate?
 
+    var isPersistent = false
```

Comment:
should this do stuff on set? i.e.:

```
didSet {
  if newValue { storeTokens } else { clearTokens }
}

?

Reply:
After merging the other PR that makes `User` responsible of managing the storage of the tokens, `isPersistent` is now a private state

Reply:
sweet! Makes more sense